dynamics/drag_dynamics.py

```diff
@@ -14,7 +14,7 @@ def drag_dynamics(x: np.ndarray, drag_const: float, latest_epoch: Epoch) -> np.n
     Computes the drag acceleration.
 
     :param x: state vector
-    :param drag_const: Drag constant in m^2/kg, calculated as -0.5 * CD * AREA / MASS
+    :param drag_const: Drag constant calculated as -0.5 * CD * AREA / MASS
```

Comment:
I think this was an accident?
```suggestion
    :param drag_const: Drag constant in m^2/kg, calculated as -0.5 * CD * AREA / MASS
```